iothub/device/src/Message.cs

```diff
@@ -254,6 +254,16 @@ public string ContentEncoding
             set => SystemProperties[MessageSystemPropertyNames.ContentEncoding] = value;
         }
 
+        /// <summary>
+        /// The digital twin component name, from where the telemetry message has originated.
+        /// This is relevant only for digital twin enabled devices.
```

Comment:
Is this wording right? "digital twin enabled"? Or certified or may be there's better one

Reply:
We have been calling them pnp-enabled devices, but yes, I do like "certified".